2024/survey.yaml

```diff
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or have contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't plan to contribute.
+      - Everything I want to contribute to is already being worked on and I don't feel the need to get involved
+      - I would like to contribute but I don't know how to get started.
+      - My experience contributing was as expected.
+      - I tried to contribute something but found it hard to get started.
+      - I tried to contribute, but received unhelpful feedback.
+      - I want to get involved in something being worked on, but don't know how to start
+      - I can't contribute because my company forbids it
+      - I was able to contribute on my own.
+      - I was able to contribute with assistance from others in the community.
```

Comment:
```suggestion
      - I don't plan to contribute.
      - Everything I want to contribute to is already being worked on and I don't feel the need to get involved.
      - I would like to contribute but I don't know how to get started.
      - My experience contributing was as expected.
      - I tried to contribute something but found it hard to get started.
      - I tried to contribute, but received unhelpful feedback.
      - I want to get involved in something being worked on, but don't know how to start.
      - I can't contribute because my company forbids it.
      - I was able to contribute on my own.
      - I was able to contribute with assistance from others in the community.
```